es/introduction.md

```diff
@@ -48,25 +48,25 @@ El código público está en el centro de las instituciones públicas modernas,
 
 El software público debe por tanto ser:
 
-* transparente
-* responsable
-* entendible por sus constituyentes
+* Transparente.
+* Responsable.
+* Entendible por sus constituyentes.
 
-Debe de reflejar los valores de la sociedad a la que sirve. Por citar un caso: siendo inclusivo y no discriminatorio.
+Debe reflejar los valores de la sociedad a la que sirve. Por citar un caso: siendo inclusivo y no discriminatorio.
 
-La mayoría de los sistemas de software propietario utilizados actualmente por las organizaciones públicas no cumplen estos requisitos. El código público -el software construido para funcionar con y como infraestructura pública, junto con los acuerdos para su producción- sí lo hace.
+La mayoría de los sistemas de software propietario utilizados actualmente por las organizaciones públicas no cumplen estos requisitos. El código público - el software construido para funcionar con y como infraestructura pública, junto con los acuerdos para su producción - sí lo hace.
 
 ### Valores del código público
 
 Consideramos que el código público ha de tener los siguientes valores centrales:
 
-* Inclusivo
-* Usable
-* Abierto
-* Legible
-* Responsable
-* Accesible
-* Sostenible
+* Inclusivo.
+* Usable.
+* Abierto.
+* Legible.
+* Responsable.
+* Accesible.
+* Sostenible.
```

Comment:
Capital letters and punctuation like the original Standard, although I like your suggestion